core/templates/pages/header_css_libs.html

```diff
@@ -46,6 +46,10 @@
 <link rel="stylesheet" type="text/css" media="screen"
       href="/templates/css/oppia.css">
 
+<!-- Importing the charts.min.css of charts.css library for visualizing the bar-charts in about page -->
```

Comment:
Instead of duplicating charts.min.css in our codebase like this, why can't you import it from e.g. third_party/static like some of the other stuff in this file?

I know you are following the material example but as a general rule we shouldn't be replicating third-party stuff wholesale within our codebase (e.g. due to different licensing requirements). Better if you import it as a 3P library similar to how we do with the other ones, and then you also won't need the styleignore thing as well.

Reply:
Hi @AkashPaloju, thanks for the great investigation and writeup. I have one question: where do we invoke the usage of postcss-calc in Oppia?

Reply:
@AkashPaloju The yarn files don't really tell us anything about the usage of postcss in the codebase ... postcss has to actually be used. I did some reading on MiniCssExtractPlugin and css loaders in webpack, and it seems that there are multiple alternatives for loaders -- css-loader, postcss-loader, gulp, etc. PostCSS setup has to be explicitly loaded in webpack.config.js -- see https://github.com/postcss/postcss#usage

So I am a bit confused. It doesn't look like we're using postcss from the webpack config. So how is this even getting used?

Reply:
You could try asking him, yup, but you could also look further down your stacktrace to see where the call originates.

Reply:
Can you provide the entire stack trace?